Dockerfile

```diff
@@ -1,20 +1,25 @@
 FROM python:3.8.5
 
-ARG MYBRANCH=master
+ARG MYBRANCH=main
 ARG COMMIT="unspecified to docker build"
 
 RUN apt-get update && \
     apt-get install -y \
+      curl \
       git \
       libgl1-mesa-glx \
-      libgtk2.0-dev
+      libgtk2.0-dev && \
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_HOME=/poetry python -
 
-RUN git clone https://github.com/AllenInstitute/ophys_nway_matching
+ENV PATH="/poetry/bin:${PATH}"
 
-RUN cd ophys_nway_matching && \
+RUN git clone https://github.com/AllenInstitute/ophys_nway_matching && \
+    cd ophys_nway_matching && \
     git checkout ${MYBRANCH} && \
-    pip install .
+    poetry install
```

Comment:
Will external users be able to use `pip` to install or are they also forced to install `poetry` to install and use this repository?

Reply:
poetry will [publish to pypi](https://python-poetry.org/docs/libraries/#publishing-to-pypi)
so, it will be pip-installable from there.
if wanting to clone and do things like `pip install -e .` `poetry install` is like this (editable)
I _think_ one can invoke the virutalenv like `poetry shell` or just run commands through `poetry run` (kind of like we do with `conda run`